### ISSUE TITLE
Don't work with Sierra 10.12.3,MBP early '11

### ISSUE THREAD

**Issue by Gimno** [dokterdok/Continuity-Activation-Tool#382]
Hello,

the Tool don't work with Sierra 10.12.3 on MacBook Pro early '11.
I use the beta from the Tool,before the last update its work.

**Comment by AussieInAZ** [comment]
I agree.  It was working fine on 10.12.2 on my late 2009 MB using an Iogear GBU521 BT dongle but unfortunately not with 10.12.3.
I have uninstalled and reinstalled with no joy yet.
Hopefully the developer will patch an update.  I miss Airdrop

**Comment by Casuallynoted** [comment]
Same here. Able to airdrop, but nothing else. No handoff on 12.4 (Early 2011 MBP)

**Comment by AussieInAZ** [comment]
I got mine working.
I removed the BT dongle and uninstalled the activation tool.
I started the install again but firstly I selected the check compatibility option.
It came back and showed that the BT was not compatible.  I then inserted the IOGEAR GBU521 dongle and reran the compatibility check.
It showed the correct dongle so I reinstalled the activation tool and now it seems to be working.
Note that I was having trouble with Air Drop not working.  It has been working since I did the above steps and I can share pictures to and from my iPhone 6 Plus

Mine is a LATE 2009 Mac Book (not Pro version)

Good luck
Aussie
